operator/deploy/crd.yaml

```diff
@@ -30,11 +30,335 @@ spec:
+                      type: string
+                      enum:
+                        - NodePort
+                        - ClusterIP
+                        - LoadBalancer
+                        - ExternalName
+                    tolerations:
+                      type: array
+                      items:
+                        type: object
+                        additionalProperties: true
+                    verbose_mode:
+                      type: number
+                      minimum: 0
+                      maximum: 5
+                    version_label:
+                      type: string
+                    view_only_mode:
+                      type: boolean
+                external_services:
+                  type: object
+                  properties:
+                    grafana:
+                      type: object
+                      properties:
+                        auth:
+                          type: object
+                          properties:
+                            ca_file:
+                              type: string
+                            insecure_skip_verify:
+                              type: boolean
+                            password:
+                              type: string
+                            token:
+                              type: string
+                            type:
+                              type: string
+                            use_kiali_token:
+                              type: boolean
+                            username:
+                              type: string
+                        dashboards:
+                          type: array
+                          items:
+                            type: object
+                            additionalProperties: true
+                        enabled:
+                          type: boolean
+                        in_cluster_url:
+                          type: string
+                        url:
+                          type: string
+                    istio:
+                      type: object
+                      properties:
+                        istio_identity_domain:
+                          type: string
+                        istio_sidecar_annotation:
+                          type: string
+                        url_service_version:
+                          type: string
+                    prometheus:
+                      type: object
+                      properties:
+                        auth:
+                          type: object
+                          properties:
+                            ca_file:
+                              type: string
+                            insecure_skip_verify:
+                              type: boolean
+                            password:
+                              type: string
+                            token:
+                              type: string
+                            type:
```

Comment:
type should be an enum - it accepts 3 values - `basic`, `bearer`, and `none` - see: https://github.com/kiali/kiali/blob/v1.9/config/config.go#L47-L50